Packs/HPE_Service_Manager/Integrations/HPE_Service_Manager/HPE_Service_Manager.yml

```diff
@@ -605,6 +652,7 @@ script:
                 Method: method,
                 Headers: {
                     Authorization: ['Basic ' + token],
+                    Connecion: ['Closed'],
```

Comment:
What is it for?

Reply:
This is for closing the connection because there is an issue which has been highlighted by HPSM regarding connection closed. FYI, https://docs.microfocus.com/SM/9.60/Codeless/Content/webservicesguide/tasks/max_sessions_exceeded_in_web_services.htm